app/components/metrics_summary_component/_index.scss

```diff
@@ -0,0 +1,36 @@
+.app-metrics__container {
+  margin-bottom: govuk-spacing(6);
+
+  @include govuk-media-query(tablet) {
+    display: grid;
+    grid-template-columns: repeat(3, minmax(govuk-px-to-rem(100), 1fr));
+    gap: $govuk-gutter;
+  }
+}
+
+.app-metrics__big-number {
+  @include govuk-typography-responsive(19);
+  @include govuk-typography-common;
+
+  padding: 5px 0;
+  margin-bottom: govuk-spacing(2);
+
+  @include govuk-media-query(tablet) {
+    @supports (display: grid) {
+      display: flex;
+      flex-flow: column nowrap;
+      justify-content: space-between;
+
+      border-top: 5px solid $govuk-border-colour;
+      border-bottom: 5px solid $govuk-border-colour;
```

Comment:
Could we use govuk spacing rather than hardcoding our own px units here? https://design-system.service.gov.uk/styles/spacing/

Reply:
Since it's border I don't think we should use `govuk-spacing` as I think that's mainly intended for margin and padding, but we can use the $govuk-border-width variable here, so good spot!

Reply:
Done!